fireblocks/src/client.rs

```diff
@@ -136,6 +143,15 @@ impl Client {
         CreateRequestBuilder(self.clone())
     }
 
+    #[must_use]
+    pub fn asset_id(&self, id: &str) -> String {
+        match (self.test_mode, id) {
+            (true, "MATIC") => "MATIC_POLYGON_MUMBAI".to_string(),
+            (true, _) => format!("{}_TEST", id),
+            (false, _) => id.to_string(),
+        }
+    }
+
```

Comment:
There is already assets on fireblocks. Can this be `fireblocks.assets().id("MATC" | "SOL")`

Reply:
okay I am going to remove test_mode and asset_id fn from the client then.